checkPropTypes.js

```diff
@@ -7,12 +7,14 @@
 
 'use strict';
 
+var constant = require('./lib/constant')
```

Comment:
all these lines are missing semicolons.
```suggestion
var constant = require('./lib/constant');
```